src/transfer_context.cpp

```diff
@@ -141,16 +141,11 @@ void TransferContext::icall(JNIEnv* rawEnv, jobject instance, jlong jPtr, jint p
 #endif
 
     write_return_value(buffer, ret_value);
-    jni::JObject local_ref{instance};
-    local_ref.delete_local_ref(env);
```

Comment:
@piiertho could you also look closely here and the other removals in this file?
Is this change needed or irrelevant after the rebase?

Reply:
All removals are optimization. There is no need to clean local ref as we're in jvm -> cpp case.